### PR TITLE
Fix web socket connection with apollo server express

### PR DESCRIPTION
**Pull request by carelynntsai**
[Notion ticket](https://www.notion.so/uwblueprintexecs/RTU-Stage-3-Push-updates-to-CCP-dashboard-d350cbe422d446d5a45114d5b931c962)

Linked to: uwblueprint/paramedics-react#63

## Changes
- Establishes websocket for front end to connect to

## Testing
- Checkout the jl-rtu-stage2 branch and refer to this PR uwblueprint/paramedics-react#63

before opening PR
- [x] check notion ticket
- [x] run linter
- [x] go through file diff

filling out PR
- [x] descriptive title
- [x] update `notion ticket` link
- [x] fill out `Changes`
  - [ ] \(optional) add inline comments in diff
- [x] fill out `Testing`
- [ ] \(optional) add `Screenshots`
- [x] assign yourself to the PR

after opening PR
- [x] link PR to notion ticket
- [x] move ticket to `In PR`
- [ ] make sure build passes
- [ ] ping `@ps` in `#paramedics-dev`

### CODE REVIEW

**Comment by carelynntsai** [comment]
Followed these docs for this PR: https://www.apollographql.com/docs/apollo-server/data/subscriptions/#subscriptions-with-additional-middleware

**Comment by carelynntsai** [comment]
@petorsong wait why did u remove the http package lol

File: app.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+const http = require('http');
```
Comment (carelynntsai) [inline app.js:1]: im using the package here

File: app.js [hunk 2]
```diff
@@ -93,7 +94,10 @@ function application({ authStrategy, graphqlPath, sessionSecret }) {
 
   server.applyMiddleware({ app, path: graphqlPath });
 
-  return app;
+  const httpServer = http.createServer(app);
```
Comment (carelynntsai) [inline app.js:97]: and here

File: app.js (hunk 1 above)
Comment (petorsong) [reply app.js:1]: it's a native node package so you don't need to fetch it elsewhere (if you look at the [source of that package](https://www.npmjs.com/package/http), it looks hella sketchy too - it's kind of a placeholder) - i tested it on my end and it works so if you can confirm the same i can merge the rest of these changes

Comment (carelynntsai) [reply app.js:1]: Ohhh okay I see thanks!

Comment (carelynntsai) [reply app.js:1]: I'm good to merge this

**Pull request merged**